pipelinewise/cli/pipelinewise.py

```diff
@@ -256,6 +263,12 @@ def load_config(self):
         else:
             self.config = {}
 
+    def get_temp_dir(self,):
```

Comment:
Is that a valid syntax `get_temp_dir(self,)`?

Reply:
seems valid but still typo, I will send new commit to fix it

Reply:
thanks @ivan-transferwise fixed by 4de3d4aab8fcf3731520783f0492cfa96cd598bd